### PR TITLE
Doc/repo nav pages

### PR DESCRIPTION
**Pull request by stroncoso-quobis**
#### What type of PR is this?

Add one of the following kinds:
* cleanup
* documentation

#### What this PR does / why we need it:

To improve repository navigation, attract developers and define clear focus on documentation usability

#### Which issue(s) this PR fixes:

none

#### Special notes for reviewers:

Live preview of API using redoc and swagger in combination with UML pre-rendered images will significan improve the legibility of the repository and the API itself

#### Changelog input

```
task(doc): Clean and updated README files
task(doc): Include PNG render of UML diagrams
task(doc): Include readme for UML diagrams folder
```

#### Additional documentation 

n/a

### CODE REVIEW

**Review by TEF-RicardoSerr — APPROVED** [review]
Looks so good! Great job Santi!

Im approving it and we can merge it after next meeting

File: code/API_definitions/README.MD [hunk 1]
```diff
@@ -1 +1,46 @@
...
+
+## Check APIs online
+
+Use the previous links to check the DEV main version of the API. Copy & paste any raw file to check them:
+
+* ReDoc: https://redocly.github.io/redoc/
+* Swagger: https://editor.swagger.io/
+
+
+## Build HTLM from sources
```
Comment (hdamker) [inline code/API_definitions/README.MD:32]:
```suggestion
## Build HTML from sources
```

Comment (stroncoso-quobis) [reply code/API_definitions/README.MD]: Done! Thanks!

File: documentation/SupportingDocuments/README.MD
Comment (hdamker) [inline documentation/SupportingDocuments/README.MD:1]: If you delete this file, the folder will disappear in GitHub (as there is no file left in the folder).

File: documentation/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+This folder include extra documentation about the API beyond the technical OpenAPI v3 API description of the `code` folder
+
+* [MeetingMinutes](MeetingMinutes/)
+  * Check this folder for the original meeting minutes.
+  
+* [API_documentation](API_documentation/)
+  * Check this folder for extra documentation to support the API definition, UML activity diagrams, API overview description, how to use guides, and other documentation related.
+
+* [SupportingDocuments](SupportingDocuments/)
```
Comment (hdamker) [inline documentation/README.md:11]: See comment below ... you are about to delete the folder.

File: documentation/SupportingDocuments/README.MD
Comment (stroncoso-quobis) [reply documentation/SupportingDocuments/README.MD:1]: I guess that I gone a little too far, I will recover the original folder content

File: documentation/README.md (hunk 2 above)
Comment (stroncoso-quobis) [reply documentation/README.md:11]: recovered file on commit 25cce13

File: documentation/SupportingDocuments/README.MD
Comment (hdamker) [reply documentation/SupportingDocuments/README.MD:1]: Overall the PR is very helpful to understand the structure of the repository content. Thanks!

**Comment by stroncoso-quobis** [comment]
Please, @hdamker , @sushanthmavenir , @pradeepachar-mavenir , May you take a look and approve in order to merge?
Thanks

**Comment by hdamker** [comment]
@stroncoso-quobis thanks for asking, but I'm not part of the team here and don't have the bandwidth to review. Please proceed without my review.

**Review by pradeepachar-mavenir — APPROVED** [review]
The changes are fine.

**Pull request merged**